packages/lesswrong/server/vulcan-lib/mutators.ts

```diff
@@ -186,6 +188,12 @@ export const createMutator: CreateMutator = async <T extends DbObject>(createMut
       Object.assign(document, { [fieldName]: autoValue });
     }
   }
+  const timeElapsed = Date.now() - start;
+  // Technically these aren't callbacks, but for the purpose of analytics we want to treat them the same way
+  captureEvent('callbacksCompleted', {
```

Comment:
Plural? Is this for multiple hooks completing or just one). You've got a period/dot in the name here, that's not something I recall having anywhere else, not sure if Postgres will/won't like it.

Reply:
Multiple callbacks in each hook.  The hooks are all initiated with names like `comments.create.before`, etc; it's a string so I don't expect postgres to have any issues.